db_create.py

```diff
@@ -1,8 +1,8 @@
 # coding=utf-8
 
 from migrate.versioning import api
-from config import SQLALCHEMY_DATABASE_URI
-from config import SQLALCHEMY_MIGRATE_REPO
+from app.config import SQLALCHEMY_DATABASE_URI
+from app.config import SQLALCHEMY_MIGRATE_REPO
 from app import db
```

Comment:
Wouldn't these changes also be required in `db_migrate.py` and `db_upgrade.py`?

Reply:
I think so. I didn't use these scripts, so didn't make any changes to them.